pkg/apis/kubeone/types.go

```diff
@@ -301,6 +303,15 @@ type CanalSpec struct {
 	MTU int `json:"mtu,omitempty"`
 }
 
+// CiliumSpec defines the Cilium CNI plugin
+type CiliumSpec struct {
+	// Disable kube-proxy to use Cilium eBPF replacement
+	// default value is false
```

Comment:
Please make this a valid Go comment and format it a little bit better.

```suggestion
	// DisableKubeProxy disables kube-proxy to use Cilium eBPF replacement.
	// Default value is false
```